pkg/kubelet/cm/devicemanager/topology_hints_test.go

```diff
@@ -316,6 +316,39 @@ func TestGetTopologyHints(t *testing.T) {
 				},
 			},
 		},
+		{
+			description:   "Single device type, some allocated to container",
```

Comment:
This test should check the case where we are able to restore hints for a set of devices, even if that device type is no longer registered with the kubelet. As such, the name should reflect that.